src/Traverse.js

```diff
@@ -21,6 +21,10 @@ export default class ElementSearchIndex {
         return this._tokenIndex.select(type);
     }
 
+    selectTokensByValue(value: string): Array<Token> {
```

Comment:
mm, well you have the same behaviour for other methods as well. Those methods could be useful, it is pretty much analogues to CSS selector `:contains(...)`. You would rarely use it, but you might be glad it is there.